document-models/arb-ltip-grantee/schema.graphql

```diff
@@ -0,0 +1,195 @@
+scalar DateTime
+
+type ArbLtipGranteeState {
+	id: ID!
+	granteeName: String
+	grantSize: Float
+	matchingGrantSize: Float
+	grantSummary: String
+	fundingAddress: String
+	fundingType: [FundingType]
```

Comment:
Having a nullable `FundingType` for this array will cause headaches down the line — you will have to handle null states in a lot of functions. Does the type actually need to be nullable? Most likely you don't want to have null values in your list.

Reply:
What I was proposing was to do `[FundingType!]` — the entire array can still be null, and it can be empty. But if you put something in the array, that thing you put in must not be null.